libs/watt/src/lib/components/data/watt-data-table.stories.ts

```diff
@@ -58,7 +60,7 @@ export const DataTable: StoryObj<WattDataTableComponent> = {
   render: (args) => ({
     props: { ...args, ...tableStoryArgs },
     template: `
-      <watt-data-table vater inset="m" [dataSource]="dataSource">
+      <watt-data-table vater inset="m">
```

Comment:
Question: Is this right?

Reply:
Yes. The `watt-data-table` doesn't have `dataSource` Input.